enhancements/olm/single-ownnamespace-enhancement-proposal.md

```diff
@@ -0,0 +1,552 @@
+
+## Operational Aspects of API Extensions
+
+### Impact of Install Mode Extensions
+
+**ClusterExtension API Enhancement:**
+- **Architectural Impact:** The `config.inline.watchNamespace` field enables runtime install mode selection, moving from compile-time (bundle) to runtime (installation) configuration
+- **Operational Impact:**
+  - Administrators must understand namespace relationships and RBAC implications
+  - Troubleshooting requires awareness of install vs watch namespace distinctions
+  - Monitoring and alerting must account for namespace-scoped operator deployments
+
+**RBAC Resource Generation:**
+- **Architectural Impact:** Dynamic RBAC generation based on install mode creates different permission patterns for the same operator
+- **Operational Impact:**
+  - Permission debugging requires understanding of install mode impact on RBAC scope
+  - Security auditing must consider namespace-level vs cluster-level permission grants
+  - Upgrade scenarios may change RBAC scope if install mode changes
+
+
+### Impact on Existing SLIs
+
+With the removal of the install mode concept in olmv1, operator packages that want to continue to use this stop-gap feature are expected to surface configuration documentation, and call out if the `watchNamespace` parameter is a part of it, along with usage example etc. Until a broad percentage of operator pacakges, especially those that don't support `AllNamespaces` mode, take action to make such documentation avaialble, a spike in bad installation is expected.
+
+**Installation Success Rate:** 
+
+*   **RBAC Validation Complexity:** Namespace-scoped installations require more complex RBAC validation to ensure the ServiceAccount has appropriate permissions for the target namespace. RBAC misconfigurations that work in AllNamespaces mode may fail in Single/OwnNamespace modes.
+    *   Example: ServiceAccount has cluster-wide read permissions but lacks namespace-specific write permissions, causing installation to fail.
+*   **Bundle Compatibility Validation:** Additional validation layer to confirm bundles support the requested install mode. Bundles that only support AllNamespaces will fail when Single/OwnNamespace is requested.
+    *   Example: Attempting to install a bundle with `watchNamespace: "test"` when the bundle CSV only declares support for AllNamespaces install mode.
+
+**Installation Time:**
+
+*   **Extended Validation Phase:** Additional validation steps for namespace existence, accessibility, and RBAC permissions add latency to the installation process. Each namespace-scoped installation must validate the target namespace and ServiceAccount permissions.
+*   **RBAC Generation Complexity:** Converting cluster-scoped RBAC to namespace-scoped Role/RoleBinding resources requires additional processing time. Complex operators with extensive permission requirements will see increased installation duration.
```

Comment:
Where are you converting cluster scoped RBAC to namespace scoped RBAC in this process? 

The only place I saw mention of a conversion was the CSV `permissions` being translated to cluster scoped RBAC resources when installing as `AllNamespaces` mode - which is the status quo.